wordpress-mobile/WordPress-Android
# Unified Login & Sign-Up: Login library screen layouts update

Fixes #11785

Depends on #12047

This PR updates the following screens layouts on the Login Library:
- Login Email Screen
- Login Email Password Screen
- Login Magic Link Request Screen
- Login Magic Link Sent Screen
- Login 2FA Screen
- Login Site Address Screen
- Login Username Password Screen
- Signup Email Screen
- Signup Magic Link Screen

With the exception of the Login Email screen, which has a flag to switch from the old layout to the one, most changes on the other screens are pretty similar: we basically hide the secondary button, override a method to setup the toolbar title and replace its old elements with new ones, if any. 

Other than that, some screens needed a shared header component, which consisted of the user's avatar and email. That component was extracted to a common layout file and a simple helper class was created to share the logic for loading the avatar.

## To test

Navigate through each of the screens listed above and make sure its appearance is in line with the design spec (internal ref: pbArwn-mA-p2). You may also want to test the flow with the feature flag turned off.

## Notes
- We still have to add the Signup Confirmation Screen, which should be done in a PR to follow.
- Both the Jetpack flow and the Share Intent flow make use of a button inside the Login Email Screen that is not being shown on the new layout. This still needs some input from the design team and will also be addressed in a future PR.

---

PR submission checklist:
